### ISSUE TITLE
TrippLite TLNET card

### ISSUE THREAD

**Issue by 4uba** [librenms/librenms#10881]
**Please do not post bugs or ask questions**

**For help and support you can use our [Discord server](https://t.libren.ms/discord) or our [community site](https://community.librenms.org).**

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x] Please provide **ALL** info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [x] Please provide as much detail as possible.
- [x] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

Hi, please add support for TrippLite TLNET card. Please find all information below:

https://p.libren.ms/view/d01027b4
https://p.libren.ms/view/e3f60ab3
https://p.libren.ms/view/6e810b28

Basically, web interface allow to monitor lot of sensors and states, but generelly I would like to have battery status (on battery or on line), battery life or remaining, voltage of the battery, amps of load.

Thanks.

**Comment by PipoCanaja** [comment]
Have a look at:
https://github.com/librenms/librenms/pull/10239

**Comment by Jellyfrog** [comment]
This request has been moved to: https://community.librenms.org/t/11601

**Comment by murrant** [comment]
This issue has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/tripplite-tlnet-card/11601/1
